src/kOS/Binding/FlightControlManager.cs

```diff
@@ -487,11 +488,21 @@ private void WheelSteer(FlightCtrlState c)
             public void Dispose()
             {
                 Enabled = false;
+                if (steeringManager != null)
+                {
+                    steeringManager.RemoveInstance(shared.Vessel);
+                    steeringManager = null;
+                }
             }
 
             public void UpdateFlightControl(Vessel vessel)
             {
                 control = GetControllerByVessel(vessel);
+                if (steeringManager != null)
+                {
+                    steeringManager = SteeringManager.SwapInstance(shared, steeringManager);
```

Comment:
maybe, but i honestly decided that we just need to get this in and we can carve this thing up later.